support-lib/composer/Outcome_composer.hpp

```diff
@@ -0,0 +1,63 @@
+/**
+  * Copyright 2024 Snap, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *    http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+#pragma once
+
+#include "djinni_composer.hpp"
+#include "../cpp/expected.hpp"
```

Comment:
can we avoid the relative paths?

Reply:
removed relative paths.  they are not needed.  "cpp" is already in include search path.